XRTK.Anchors/Packages/com.xrtk.anchors/Runtime/Definitions/SupportedAnchorType.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace XRTK.Definitions.Anchors
+{
+    public enum SupportedAnchorType
+    {
+        None,
+        Azure_Spatial_Anchors
```

Comment:
Can't we do the same method we did to add new controller support?

That way we don't have to maintain an enum?

Reply:
I'll look in to that then.

Reply:
Switched to SystemType for more dynamisim